<launch>

    <node pkg="mpc" type="run_tracking.py" name="mpc" output="screen" respawn="true">

        <param name="publish_rate" value="40" />

        <param name="base_link_frame" value="ugr/car_base_link" />

        <param name="speed_start" value="2" />
        <param name="speed_stop" value="10" />
        <param name="distance_start" value="3" />
        <param name="distance_stop" value="5" /> 

        <param name="wheelradius" value="0.1" />    

        <remap from="/input/path" to="/pathplanning/path" />
        <remap from="/input/odom" to="/ugr/car/odometry/filtered/odom"/>
        <remap from="/output/drive_effort_controller/command" to="/ugr/car/drive_effort_controller/command" />
        <remap from="/output/drive_velocity_controller/command" to="/ugr/car/drive_velocity_controller/command" />
        <remap from="/output/steering_velocity_controller/command" to="/ugr/car/steering_velocity_controller/command" />
        <remap from="/output/steering_position_controller/command" to="/ugr/car/steering_position_controller/command" />
        <remap from="/output/target_point" to="/pure_pursuit/target_point"/>
        <remap from="/output/x_prediction" to="/mpc/x_prediction"/>
        <remap from="/output/ref_track" to="/mpc/ref_track"/>
        <remap from="output/left_line" to="/mpc/left_line"/>
        <remap from="output/right_line" to="/mpc/right_line"/>
    </node>

</launch>